Makefile

```diff
@@ -50,7 +50,7 @@ build-virtual:
 .PHONY: build-virtual-unsafe
 build-virtual-unsafe:
 	mkdir -p $(BUILD)
-	cd $(BUILD) && CC=$(CC) CXX=$(CXX) cmake -DCOMPILE_TARGET=virtual -DCMAKE_EXPORT_COMPILE_COMMANDS=1 -DVERBOSE_LOGGING=ON -DCCF_UNSAFE=ON -GNinja ..
+	cd $(BUILD) && CC=$(CC) CXX=$(CXX) cmake -DCOMPILE_TARGET=virtual -DCMAKE_EXPORT_COMPILE_COMMANDS=1 -DVERBOSE_LOGGING=ON -DCCF_UNSAFE=OFF -GNinja ..
```

Comment:
Should this be `ON` based on the name of the target?

Reply:
I've updated the name, but there isn't a virtual build of ccf with unsafe support so we have an error for when we try to use it in LSKV.